includes/admin/meta-boxes/class-wc-meta-box-order-data.php

```diff
@@ -459,8 +459,8 @@ public static function output( $post ) {
 							}
 
 							if ( apply_filters( 'woocommerce_enable_order_notes_field', 'yes' == get_option( 'woocommerce_enable_order_comments', 'yes' ) ) && $post->post_excerpt ) {
-								echo '<p><strong>' . __( 'Customer provided note:', 'woocommerce' ) . '</strong> ' . nl2br( esc_html( $post->post_excerpt ) ) . '</p>';
-							}
+                                echo '<p><strong>' . __( 'Customer provided note:', 'woocommerce' ) . '</strong> <span class="customer-provided-note-text">' . nl2br( esc_html( $post->post_excerpt ) ) . '</span></p>';
```

Comment:
Looks better now, but still not possible to merge this Pull Request, since is changing the indentation style to spaces.

Reply:
@claudiosanches well, that's wiered... I try other editor.